### DISCUSSION TITLE
[Spectre-Importer] API limit & trial

### DISCUSSION THREAD

**Discussion by hitokiri8x** [firefly-iii/firefly-iii#5211] [Firefly III] [answered]
Hello, I' haven't installed nor configured anything, just doing evaluation..
I haven't figured out what are the limit of the spectre importer moreover the validity of the account.
After 90 days I need a new email and a new account? or it's just a "push button to get the new API key"  from account panel? or 
 PAY to continue without workaround?
I read that it import only the last 90 days of transaction  but that could be bank related..  
Many thanks.

**Comment by JC5** [comment, marked as answer]
You have to ask them to be sure but their account trial limit is 90 days; after this time api keys may no longer work, not even if you replace them. I also hear this limit doesn't apply to Firefly III users but I'm not sure. 

Also 90 days is the limit for most banks, they'll not go further back. This also depends on your bank.

So no definitive answer im afraid.

**Reply by hitokiri8x** [reply]
Thanks!  Looking at your answer game me another idea to another search and I found #3760, which killed everything  xD
I think I'll fallback to selenium ( or whatever is used today ) to download a csv from my homebanking and parse automatically...

**Reply by JC5** [reply]
Keep an eye on Twitter. I'm working on a nordigen api that should remain free.

**Comment by JC5** [comment]
Oh, re-registering with a new email address may also work, but if Spectre gives you an IP ban don't look at me 😉
